Description: typo in backend/genesys.conf.in
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=743848
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Last-Update: 2014-10-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/backend/genesys.conf.in
===================================================================
--- trunk.orig/backend/genesys.conf.in
+++ trunk/backend/genesys.conf.in
@@ -2,7 +2,7 @@
 
 #
 # scanners that are not yet supported
-# uncomment them only for developpment purpose
+# uncomment them only for development purpose
 #
 
 # UMAX Astra 4500 and Avision iVina 1600
